project/project.json5

```diff
@@ -0,0 +1,238 @@
+{
+    project_id: 'US_DOE_DECARB_2023',
+    description: '',
+    name: 'U.S. DOE DECARB High Resolution Load Projections 2023',
+    datasets: [
+        {
+            dataset_id: 'decarb_2023_buildings',
+            dataset_type: 'modeled',
+            version: '1.0.0',
+            required_dimensions: {
+                single_dimensional: {
+                    sector: {
+                        base: ['res', 'com'],
+                    },
+                    subsector: {
+                        base: ['unspecified'],
+                    },
+                    metric: {
+                        supplemental: {
+                            name: 'building-end-uses-collapsed',
+                            record_ids: ['building_end_uses'],
+                        },
+                    }
+                }
+            }
+        },
```

Comment:
This specification conveys that the buildings dataset is expected to provide information either directly or through mappings for all project dimension elements except for the ones excluded by these filters on sector, subsector, and metric. That is, we expect to have (mapped) buildings data for all counties, model years, scenarios, and weather years; but only expect buildings data for a subset of the sectors ('res' and 'com'), subsectors ('unspecified'), and end-uses (project end-uses that map into 'building_end_uses').

Reply:
Thanks — this looks correct to me.